<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmAntLine</class>
 <widget class="QWidget" name="frmAntLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="AntLine" name="antLine1" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="AntLine" name="antLine2" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="AntLine" name="antLine3" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="AntLine" name="antLine4" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AntLine</class>
   <extends>QWidget</extends>
   <header>antline.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
